Cargo.toml

```diff
@@ -12,6 +12,26 @@ homepage = "https://github.com/alloy-rs/op-alloy"
 repository = "https://github.com/alloy-rs/op-alloy"
 exclude = ["benches/", "tests/"]
 
+[workspace.dependencies]
+# Alloy
+alloy-primitives = { version = "0.7.0", default-features = false }
+alloy = { git = "https://github.com/alloy-rs/alloy", rev = "39b8695", features = [
```

Comment:
Make sure to update to 0.7.1 and a recent commit

In the examples repo we have an test that performs an integration test against the latest commit on a nightly basis: https://github.com/alloy-rs/examples/blob/main/.github/workflows/integration.yml which may be useful